index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <a class="name">Muh Noman</a>
+      <nav>
+        <img class="navi" src="/images/navi.png" alt="navigation" />
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h1>Hey There. I’m Noman</h1>
+        <h6>I’am a Software Developer</h6>
```

Comment:
- [ ] You should to use the `Crete Round` font-family in the Figma design as was suggested by the previous reviewer. I have read your your comment in this thread and I think this [freecodecamp article](https://www.freecodecamp.org/news/how-to-use-google-fonts-in-your-next-web-design-project-e1ad48f1adfa/) on how to add google fonts can help you resolve the issue.

Reply:
I Applied these changes now thanks for your suggestion